package.json

```diff
@@ -124,6 +120,13 @@
     "vega-util": "~1.10.0",
     "yargs": "~13.2.4"
   },
+  "beemo": {
```

Comment:
move this below peer dependencies

Reply:
We should actually just remove this since it should be part of the dev config deps.